### DISCUSSION TITLE
What is the bundle size of Semi UI?

### DISCUSSION THREAD

**Discussion by iamsahebgiri** [DouyinFE/semi-design#475] [Q&A] [answered]
I was trying out Semi UI but I couldn't able to find out its bundle size (minified and gzipped).  I have tried using [Bundlephobia](https://bundlephobia.com/package/@douyinfe/semi-ui) but it says  install error may be because of SCSS not sure.

**Comment by pointhalo** [comment, marked as answer]
bundle size：

![image](https://user-images.githubusercontent.com/88709023/147720565-c16b1503-1e0f-43c6-b948-a8aaa81baf6a.png)

Also you can access files from unpkg to get these information.
https://semi.design/en-US/start/getting-started
![image](https://user-images.githubusercontent.com/88709023/147720747-05b06e20-8017-4c36-ae59-4e742caab95d.png)

**Reply by iamsahebgiri** [reply]
Correct me if I am wrong, semi UI is tree shakable so depending on the components we use, we will get less size than 361.2Kb provided that I use bundler.

**Reply by pointhalo** [reply]
yes
